packages/components/pages/f-contact-preferences/src/tenants/en-GB.js

```diff
@@ -1,8 +1,17 @@
 const messages = {
     errorMessages: {
-        errorHeading: 'Error',
-        pageLoad: {
+        loading: {
+            heading: 'Error',
             description: 'Sorry, there was a problem fetching your contact preferences. Please try again later.'
+        },
+        saving: {
+            heading: 'Something went wrong',
+            description: 'Your changes have not been saved, please try again.'
```

Comment:
Out of interest, who's providing the messaging for this? Main reason I ask is that this tone of error message is really frustrating for users if they just see this every time they try submitting and it's not their fault. Ideally, the message should try and say why something went wrong and what to do if it keeps happening.

Not necessarily something that needs updating in this PR, but for me, it should say something more along the lines of `There was an issue saving your preferences, please try submitting them again. If this issue persists, please contact our help centre/contact us blah blah`. Also `Something went wrong` as a heading for an error is also pretty useless as a user 😆 

Imagine we're taking this from current Contact Preferences messages, but would be good to flag to improve this experience in a future ticket.

Reply:
Took text from here https://www.figma.com/file/GrGeJcVgsgyEJ31SNvPR3r/Web---Account?node-id=555%3A10198

Reply:
@ashleynolan Who should I approach in the business for different wording?

Reply:
@oliversweb You can ask in the "product copy requests" channel